downstream/assemblies/eda/assembly-installation-eda-controller.adoc

```diff
@@ -0,0 +1,19 @@
+ifdef::context[:parent-context: {context}]
+
+[id="installation-of-eda-controller"]
+
+= Installation of {EDAname} controller
+
+The installation of Event-Driven Ansible controller coincides with other Ansible Automation Platform components. Like automation controller and automation hub, the setup includes default settings for specific variables in the inventory files.
+
+include::eda/ref-installing-eda-controller-on-aap.adoc[leveloffset=+1]
+include::eda/ref-deploy-eda-controller-with-aap-operator-on-ocp.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+== Additional resources (or Next steps)
```

Comment:
Change this to a . heading to be consistent with rest of the doc:
.Additional resources (or Next steps)